### PR TITLE
STAY-3924 added new method to get JsonTranslations

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/Models/Translation.php [hunk 1]
```diff
@@ -52,4 +53,8 @@ public function scopeSelectDistinctGroup($query)
         return $query->select(DB::raw($select));
     }
 
+    public function translationsJson(){
+    	return $this->hasMany(TranslationsJson::class, 'key_php', 'key');
+	}
```
Comment (madu2789) [inline src/Models/Translation.php:58]: 💡 mal formateo?

File: src/Controller.php [hunk 2]
```diff
@@ -135,6 +143,35 @@ public function postPublish($group = null)
...
 
-        return ['status' => 'ok'];
-    }
+		return ['status' => 'ok'];
+	}
+
+	/**
+	 * @return array
+	 */
+	public function postSaveKeyJson()
```
Comment (madu2789) [inline src/Controller.php:152]: 💡 return typehinting

**Review by madu2789 — APPROVED** [review]
Todo ok!

File: src/Controller.php (hunk 2 above)
Comment (BlackfireSFL) [reply src/Controller.php]: must be support php 5.6, be careful with the type hinting

Comment (stayforlong) [reply src/Controller.php:152]: must be support php 5.4, be careful with the type hinting.

File: src/Models/Translation.php (hunk 1 above)
Comment (ghost) [reply src/Models/Translation.php:58]: no consigo arreglarlo 🤔

**Pull request merged**